### PR TITLE
Blog post on whats new in Camel 3.10.

### PR DESCRIPTION
**Pull request by davsclaus**
Feedback is welcome.

Fix any typos, or improve the reading of the text is welcome, and you are welcome to do that and update the PR

### CODE REVIEW

File: content/blog/2021/05/Camel310-Whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+in a single JAR which otherwise would cause Camel metadata files to be overridden.
+
+### Debezium
+
+Camel now supports Debezium 1.5.
+
+### Quarkus
+
+This release is preparing for supporting the upcoming Quarkus 2.0 release.
+Camel is now upgraded to support VertX and the MicroProfile v4 specification.
```
Comment (apupier) [inline content/blog/2021/05/Camel310-Whatsnew/index.md:103]:
```suggestion
Camel is now upgraded to support Vert.X and the MicroProfile v4 specification.
```

**Review by oscerd — APPROVED** [review]
I would mention the s3 streaming upload feature too. Looks good overall

**Comment by davsclaus** [comment]
@oscerd do you want to add something about that s3 streaming upload thing? If so you are welcome to update this PR, or add a comment here, then I can do it.

**Comment by oscerd** [comment]
@davsclaus thanks, go ahead and merge, there is a blog post published before. We can eventually link it. Thanks

**Pull request merged**